src/BookReader/Mode1UpLit.js

```diff
@@ -1,6 +1,6 @@
 // @ts-check
-import { customElement, html, LitElement, property, query } from 'lit-element';
-import { styleMap } from 'lit-html/directives/style-map';
+import { customElement, html, LitElement, property, query } from 'lit/decorators.js';
```

Comment:
Only `property` and `customElement` are from `lit/decorators.js`; the other ones are from `lit`;

```suggestion
import { html, LitElement, query } from 'lit';
import { customElement, property } from 'lit/decorators.js';
```